### PR TITLE
Russian localization

### PR DESCRIPTION
**Pull request by e-shl**
When translating, there were noticed flaws / bugs in the interface
<details>

</details>
Part managed to fix the translation adaptation
<details>

image
</details>

Please fix!

### CODE REVIEW

**Comment by e-shl** [comment]
> When translating, there were noticed flaws / bugs in the interface
> 
> Part managed to fix the translation adaptation
> Please fix!

**Comment by Jays2Kings** [comment]
Some of these bugs have been fixed already and one new string has been added since this pr was made ("show all categories")

I'll fix the other issues mentioned in soon

**Comment by e-shl** [comment]
@Jays2Kings, done.

**Comment by Jays2Kings** [comment]
It leaves an error (but still compiles) if you remove the %1$s when the main string has one.

Is it possible to use Сбросить все главы для этой манги (taken from the main app's translation but replace манги with %1$s ?

**Comment by e-shl** [comment]
@Jays2Kings It works without errors. BUT, you can fix interface problems, at least

**Comment by Jays2Kings** [comment]
It's already fixed @e-shl it is now scrollable

Also I meant reset_chapters_for_this strings leaves an error since you removed the %1$s, I was asking if you can reword it to include it

**Comment by e-shl** [comment]
@Jays2Kings me create a new pull requests to return %1$s ?

**Comment by Jays2Kings** [comment]
If you please, thanks
